src/braket/circuits/braket_program_context.py

```diff
@@ -66,7 +66,7 @@ def add_phase_instruction(self, target: tuple[int], phase_value: float) -> None:
         instruction = Instruction(BRAKET_GATES["gphase"](phase_value))
         self._circuit.add_instruction(instruction)
 
-    def add_gate_instruction(
+    def add_gate_instruction(  # noqa: D417
```

Comment:
Shouldn't this be fixed? Or at least add a TODO.

Reply:
Yes, I am taking all of these inline ignores as TODOs in a separate PR.